Source/Core/Core/IOS/ES/TitleManagement.cpp

```diff
@@ -25,8 +25,11 @@ namespace IOS::HLE
 static ReturnCode WriteTicket(FS::FileSystem* fs, const ES::TicketReader& ticket)
 {
   const u64 title_id = ticket.GetTitleId();
+  std::string path = Common::GetTicketFileName(title_id);
```

Comment:
Consider:
```suggestion
  const std::string path = ticket.IsV1Ticket() ? Common::GetV1TicketFileName(title_id) : Common::GetTicketFileName(title_id);
```
so we can keep `path` const and avoid having to do a useless string allocation + formatting + copy + deallocation.

Might be worth having a helper function that takes in a ticket and returns the ticket file name based on `ticket.GetTitleId()` and `ticket.IsV1Ticket()`.